app/views/subscription.scala.html

```diff
@@ -48,7 +48,7 @@
 
   @formHelper(action = controllers.routes.SubscriptionController.continue) {
 
-    <div class="govuk-form-group" id="isAgent">
+    <div class="govuk-form-group" id="f-0">
```

Comment:
Can't find any examples of this being used in other repos ?
Functionally it works which is great - just wondering if we should be using something else

Reply:
removed the id="f-0" and still functions as required - resolved